src/Hash.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef HASH_HPP
+#define HASH_HPP
+
+#include <napi.h>
+
+#include "xxhash/xxhash.h"
+
+namespace nbs {
+
+    namespace Hash {
```

Comment:
I would keep this lowercase to prevent confusion with it being a class
```suggestion
    namespace hash {
```